jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -490,15 +553,15 @@ public String getAgentHost() {
         return null;
       }
 
-      return this.senderConfiguration.agentHost;
+      return this.senderConfiguration.builder.agentHost;
```

Comment:
Possibly we should deprecate these accessors (getAgentHost/getAgentPort) as looks like they are only used in tests?

Reply:
I was thinking about this, We definitely need accessors like `Configuration.getReporterConfiguration` then `getSenderConfiguration` to be able to override sender without re-creating other configs.

I am wondering whether there is an use case when app gets the configuration from env and then changes behaviour - e.g. if used endpoint it will get it and override to https for example.

Reply:
Don't think we should be supporting such use cases - but if someone wanted to make a case for it later, it could be discussed then.